src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -163,7 +162,7 @@ const bottomOverlayStyles = StyleSheet.create({
     borderRadius: 16
   },
   nextButton: {
-    backgroundColor: COLORS.blueOnboarding,
+    backgroundColor: COLORS.blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the next button has been changed from `COLORS.blueOnboarding` to `COLORS.blue`. Make sure this change aligns with the design specifications.
```diff
-    backgroundColor: COLORS.blueOnboarding,
+    backgroundColor: COLORS.blue,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->